planner/core/logical_plans.go

```diff
@@ -246,6 +251,10 @@ type LogicalAggregation struct {
 	// groupByCols stores the columns that are group-by items.
 	groupByCols []*expression.Column
 
+	// hintInfo stores the aggregation algorithm hint information specified by client.
+	hintInfo      *tableHintInfo
```

Comment:
Not needed since we have `preferAggType`?

Reply:
Got it.